crates/mysten-network/src/client.rs

```diff
@@ -147,9 +148,9 @@ impl MyEndpoint {
                 let https = hyper_rustls::HttpsConnectorBuilder::new()
```

Comment:
Before we land this can we also make sure to set tcp_nodelay? https://docs.rs/hyper-util/latest/hyper_util/client/legacy/connect/struct.HttpConnector.html#method.set_nodelay